src/Uno.UI.Composition/Composition/CompositionEffectBrush.skia.cs

```diff
@@ -0,0 +1,1726 @@
+									{
+										return sourceFilter1;
+									}
+									else if (crossfade >= 1.0f)
+									{
+										return sourceFilter2;
+									}
+
+									SKImageFilter fbFilter = SKImageFilter.CreateColorFilter(
+										SKColorFilter.CreateColorMatrix(
+											new float[]
+											{
+												crossfade, 0,         0,         0,         0,
+												0,         crossfade, 0,         0,         0,
+												0,         0,         crossfade, 0,         0,
+												0,         0,         0,         crossfade, 0
+											}),
+										sourceFilter2);
+
+									string shader = $@"
+											uniform shader input;
+											uniform half crossfade;
+
+											half4 main() 
+											{{
+												half4 inputColor = sample(input);
+												return inputColor - (inputColor * crossfade);
+											}}
+										";
+
+									SKRuntimeEffect runtimeEffect = SKRuntimeEffect.Create(shader, out string errors);
+									if (errors is not null)
+									{
+										return null;
+									}
+
+									SKRuntimeEffectUniforms uniforms = new(runtimeEffect)
+									{
+										{ "crossfade", crossfade }
+									};
+									SKRuntimeEffectChildren children = new(runtimeEffect)
+									{
+										{ "input", null }
+									};
+
+									SKImageFilter amafFilter = SKImageFilter.CreateColorFilter(runtimeEffect.ToColorFilter(uniforms, children), sourceFilter1);
+
+									return SKImageFilter.CreateBlendMode(SKBlendMode.Plus, fbFilter, amafFilter, new(bounds));
+								}
+
+								return null;
+							}
+						case EffectType.LuminanceToAlphaEffect:
+							{
+								if (effectInterop.GetSourceCount() == 1 && effectInterop.GetSource(0) is IGraphicsEffectSource source)
+								{
+									SKImageFilter? sourceFilter = GenerateEffectFilter(source, bounds);
+									if (sourceFilter is null && !_isCurrentInputBackdrop)
+									{
+										return null;
+									}
+
+									_isCurrentInputBackdrop = false;
+
+									return SKImageFilter.CreateColorFilter(SKColorFilter.CreateLumaColor(), sourceFilter, new(bounds));
+								}
+
+								return null;
+							}
+						case EffectType.LinearTransferEffect:
+							{
+								if (effectInterop.GetSourceCount() == 1 && effectInterop.GetPropertyCount() == 13 && effectInterop.GetSource(0) is IGraphicsEffectSource source)
+								{
+									SKImageFilter? sourceFilter = GenerateEffectFilter(source, bounds);
+									if (sourceFilter is null && !_isCurrentInputBackdrop)
+									{
+										return null;
+									}
+
+									_isCurrentInputBackdrop = false;
+
+									if (Compositor.IsSoftwareRenderer is true)
+									{
+										if (sourceFilter is not null)
+										{
+											return sourceFilter;
+										}
+										else
+										{
+											return SKImageFilter.CreateOffset(0, 0, null, new(bounds));
+										}
+									}
+
+									effectInterop.GetNamedPropertyMapping("RedOffset", out uint redOffsetProp, out _);
```

Comment:
Are we able to use `nameof(RedSlope)` somehow? Or maybe convert those in actual property getters?

Reply:
we can't do this here, Windows Composition Effects is designed to be separate from effect description types (such as Win2D effect types) because users can provide their own effect descriptors by implementing IGraphicsEffectD2D1Interop

So we can either use property names or their indices, Windows internally uses indices but I picked names for readability.
maybe we can create a struct with const strings (or indices) for these properties like I did for the AcrylicBrush impl?